docs/docs/getting-started/installing-nebari.md

```diff
@@ -7,6 +7,7 @@ description: A guide to help you install Nebari for your team.
 
 This installation guide provides the basic instructions to install and deploy Nebari for the first time, and assumes you are already familiar with the [Conda](https://docs.conda.io/projects/conda/en/latest/) and [Python packaging](https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-packages) ecosystems.
 
+# Installing the Nebari package
```

Comment:
This title feels out of place - should this be bumped to the top of the page?

Reply:
Fixed.